extern/CMakeLists.txt

```diff
@@ -79,6 +79,10 @@ if ("${CMAKE_SYSTEM_NAME}" MATCHES "Linux")
     set(SRCS ${SRCS} nativefiledialog/src/nfd_zenity.c)
 endif()
 
+if ("${CMAKE_SYSTEM_NAME}" MATCHES "OpenBSD")
```

Comment:
Could you combine this with the linux block using AND instead of copying it?

Reply:
Yeah, that's fine. I originally had it that way but then separated it because when the other BSDs come around it would become an unwieldy long long.

And yes, it does work.